app/controllers/organizations/staff/adoption_application_reviews_controller.rb

```diff
@@ -41,6 +41,10 @@ def update
     end
   end
 
+  def show
```

Comment:
I think we should add the same authorization check as we do on the index. You'll just need to call it in the Show action and add an alias action in `Organizations::AdopterApplicationPolicy`

Reply:
Done.. Added authorization for the show action